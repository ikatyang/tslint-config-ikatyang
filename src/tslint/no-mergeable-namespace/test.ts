// good

namespace good1 {
  let something1;
  let something2;
}

// bad

namespace bad1 {
  let something1;
}

namespace bad1 {
  let something2;
}
