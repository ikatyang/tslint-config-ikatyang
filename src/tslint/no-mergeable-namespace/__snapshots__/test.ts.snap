// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

namespace good1 {
  let something1;
  let something2;
}

// bad

namespace bad1 {
  let something1;
}

namespace bad1 {
          ~~~~   [Mergeable namespace 'bad1' found. Merge its contents with the namespace on line 10.]
  let something2;
}

`;
